docs/overview.md

```diff
@@ -0,0 +1,82 @@
+# Overview
+
+### Terminology
+- **Users** - end-users that want to transfer HBAR or HTS tokens from Hedera to EVM-based chain or Wrapped HBAR and Wrapped Tokens from EVM-based chain to Hedera
+- **Token Developers** - the developers of HTS tokens
+- **Validators** - parties/entities that are running the Validator node. They are providing authorisation for the minting and burning of wrapped tokens on the EVM-based chain as-well as transferring wrapped tokens back to Hedera.
+- **WHBAR** - ERC20 token issued and operated by Bridge validators. The token represents "wrapped" HBAR on the EVM-based chain. In other words we can say that Hbar is the `native` asset and `WHBAR` is the `non-native` asset.
+- **WHTS** - ERC20 token issued and operated by Bridge validators. The token represents "wrapped" HTS token on the EVM-based chain. In this case, `HTS` token is a `native` one and `WHTS` token is `non-native`.
```

Comment:
Divide into:
* Actors:
    * Users
    * Developers
    * Validators
* Asset - description
    * WHBAR (even add HBAR)
    * WHTS (even add HTS)

Maybe here would be a better place to explain what are bridge accounts, fee accounts, etc., in case someone wants to skip the governance step